.github/workflows/swift.yml

```diff
@@ -32,6 +32,7 @@ on:
       - main
       - rc/*
       - codeql-cli-*
+  workflow_dispatch:
```

Comment:
It's really handy to be able to test these workflows (especially given that they have branch constraints that prevent normal testing)